public/app/features/datasources/settings/HttpSettingsCtrl.ts

```diff
@@ -6,6 +6,7 @@ coreModule.directive('datasourceHttpSettings', () => {
       current: '=',
       suggestUrl: '@',
       noDirectAccess: '@',
+      showForwardOAuthIdentityOption: '@',
```

Comment:
I'm not 100% sure on this as my angular knowledge is quite limited but if we add a property to the `scope` here do we also need to add it to the template [here](https://github.com/grafana/grafana/blob/hide-oauth-forward/public/app/features/datasources/partials/http_settings_next.html#L1)?

Reply:
I also have limited knowledge about Angular but I think you're right Jack and furthermore I think we need to add this property to [`angular_wrappers`](https://github.com/grafana/grafana/blob/hide-oauth-forward/public/app/core/angular_wrappers.ts#L165) as well.